test/blackbox-tests/test-cases/missing-loc-run/run.t

```diff
@@ -3,6 +3,9 @@ Exact path provided by the user:
   $ dune runtest --root precise-path
   Entering directory 'precise-path'
   File "dune", line 1, characters 0-49:
+  1: (alias
```

Comment:
The case where the loc is "File X, line 1, characters (the whole file)" is a bit special, because it corresponds to `Loc.in_file`, which means that there's no precise part that corresponds to this error.

Do you think that we should display the whole file in that case?

Reply:
I'd lean towards not displaying the whole file, since we aren't able to give an accurate location

Reply:
I'm also against displaying the huge file as it can be huge. There are some generated dune files out there.

Reply:
@rgrinberg @emillon How about we treat this scenario the same as a long stanza? I added a check that I only print out content if its less than 10 lines. I was currently not displaying anything if the stanza is over 10 lines, but we could display the 10 lines (or any other number) and then stop.

Reply:
I have pushed an update addressing all the other comments. i'll be happy to update again if you want this behaviour changed

Reply:
I think displaying the first N lines should be a decent option. If its okay I will keep the current behavior of not displaying anything if over 10 lines. I will make a separate PR which deals with displaying a truncated view of the file.

Reply:
:+1: